lib/list.js

```diff
@@ -0,0 +1,85 @@
+const Id = require('hypercore-id-encoding')
+const c = require('compact-encoding')
+const Hyperbee = require('hyperbee')
+const same = require('same-object')
+
+const types = ['core', 'bee', 'drive']
+
+module.exports = class List {
+  constructor (core) {
+    this.core = core
+    this.bee = new Hyperbee(core, { keyEncoding, valueEncoding })
+  }
+
+  ready () { return this.bee.ready() }
+  close () { return this.bee.close() }
+
+  async put (key, opts = {}) {
+    const id = Id.normalize(key)
+    if (types.indexOf(opts.type) === -1) throw new Error('Invalid type: ' + opts.type)
+
+    return this.bee.put(id, {
+      type: opts.type,
+      description: opts.description || '',
+      seeders: !!opts.seeders
+    }, { cas })
+  }
+
+  async update (prev, opts = {}) {
+    return this.put(prev.key, Object.assign({}, prev.value, opts))
+  }
```

Comment:
this doesn't belong here imo

Reply:
Renamed to `edit`. In any UI (menu or app) it should use `entries` method to get full kv entries and show them, and later `edit` method to easily change them, plays really well